custom/icds_reports/tasks.py

```diff
@@ -1199,6 +1199,46 @@ def build_incentive_report(agg_date=None):
         agg_date = current_month - relativedelta(months=1)
     for state in state_ids:
         AWWIncentiveReport.aggregate(state, agg_date)
+
+    awc_performance_rows_list = list(AWWIncentiveReport.objects.values_list('awc_id', 'is_launched',
+                                                                            'valid_visits', 'visit_denominator',
+                                                                            'wer_weighed', 'wer_eligible',
+                                                                            'incentive_eligible')
+                                     .order_by('awc_id'))
+    awc_aggregate_rows_list = list(AggAwc.objects.values_list('awc_id', 'is_launched',
+                                                              'valid_visits', 'expected_visits')
+                                   .order_by('awc_id'))
+    is_launched_check_bad_data = []
+    home_conduct_check_bad_data = []
+    eligibility_check_bad_data = []
+    for index, awc_from_performance in enumerate(awc_performance_rows_list):
+        awc_from_aggregate = awc_aggregate_rows_list[index]
+
+        # check for launched AWCs
+        row_data = get_awc_is_launched_mismatch_row(awc_from_performance, awc_from_aggregate)
```

Comment:
These checks are very repetitive, it would be nice to pull this out into a function